collect_app/src/main/java/org/odk/collect/android/external/ExternalSQLiteOpenHelper.java

```diff
@@ -263,4 +265,23 @@ private void onProgress(String message) {
             formLoaderTask.publishExternalDataLoadingProgress(message);
         }
     }
+
+    /**
+     * replaceByteOrderMark
+     *
+     * Removes the presence of a Byte Order Mark from the initial UTF-8 String
+     *
+     * Fixes issue: https://github.com/opendatakit/collect/issues/1695
+     *
+     * @param firstLine String to scan for a Byte Order Marker
+     * @return String with any Byte Order Marker removed
+     */
+    private String replaceByteOrderMark(String firstLine) {
+        final String BOM = "\uFEFF";
```

Comment:
I would recommend removing this variable since you added a good comment above and it's used only in one place below (you can just use `\uFEFF`).

Reply:
I only put it there for code readability purposes, but yes it can be removed.

Reply:
I think it's clear enough thanks to your comment above.